packages/types/src/sdk/events/backup.ts

```diff
@@ -1,7 +1,13 @@
 import { BaseEvent } from "./event"
+import { AppBackupTrigger, AppBackupType } from "../../documents"
 
 export interface AppBackupRestoreEvent extends BaseEvent {
   appId: string
-  backupName: string
   backupCreatedAt: string
 }
+
+export interface AppBackupTriggeredEvent extends BaseEvent {
+  appId: string
```

Comment:
We should actually add the backup id as well, most other events have the primary id. It's always useful to have the id of the event triggering, as down the line we may use these events to trigger webhooks. On the consuming side if you have the id of the thing that triggered the event you can query for it over the API.